src/GuppyDataExplorer/ExplorerTable/index.jsx

```diff
@@ -1,6 +1,6 @@
-import React from 'react';
+import React, { useEffect, ReactNode } from 'react';
```

Comment:
why importing `ReactNode` here?

Reply:
I think here we can remove both, I don't see `useEffect` anywhere either.